views/home.tmpl

```diff
@@ -186,73 +186,185 @@
-                            <th>Time ({{timezone}})</th>
+                        <tr class="card-theader">
+                            <th class="text-left pl-3">Height</th>
+                            <th><div class="position-relative text-center" data-tooltip="regular transactions">R</div></th>
+                            <th><div class="position-relative text-center" data-tooltip="votes">V</div></th>
+                            <th><div class="position-relative text-center" data-tooltip="tickets">T</div></th>
+                            <th><div class="position-relative text-center" data-tooltip="revocations">X</div></th>
+                            <th><div class="position-relative text-center" data-tooltip="total DCR sent">DCR</div></th>
+                            <th class="d-none d-sm-table-cell d-md-none d-lg-table-cell"><div class="position-relative text-center">Size</div></th>
+                            <th class="text-right pr-3" data-target="time.header" data-jstitle="Age">Time ({{timezone}})</th>
                         </tr>
                     </thead>
                     <tbody data-target="blocklist.table">
                         {{range .Blocks}}
                         <tr data-height="{{.Height}}" data-link-class="fs18">
-                            <td data-type="height"><a href="/block/{{.Height}}" class="fs18">{{.Height}}</a></td>
-                            <td data-type="tx">{{.Transactions}}</td>
-                            <td data-type="votes">{{.Voters}}</td>
-                            <td data-type="tickets">{{.FreshStake}}</td>
-                            <td data-type="revocations">{{.Revocations}}</td>
-                            <td data-type="size">{{.FormattedBytes}}</td>
-                            <td data-type="age" class="jsonly" data-target="time.age" data-age="{{.BlockTime}}"></td>
-                            <td data-type="time">{{.BlockTime}}</td>
+                            <td class="text-left pl-2" data-type="height"><a href="/block/{{.Height}}" class="fs18">{{.Height}}</a></td>
+                            <td class="text-center" data-type="tx">{{.Transactions}}</td>
+                            <td class="text-center" data-type="votes">{{.Voters}}</td>
+                            <td class="text-center" data-type="tickets">{{.FreshStake}}</td>
+                            <td class="text-center" data-type="revocations">{{.Revocations}}</td>
+                            <td class="text-center" data-type="value">{{threeSigFigs .Total}}</td>
+                            <td class="text-center d-none d-sm-table-cell d-md-none d-lg-table-cell" data-type="value">{{.FormattedBytes}}</td>
+                            <td class="text-right pr-2" data-type="age" data-target="time.age" data-age="{{.BlockTime}}">{{.BlockTime}}</td>
                         </tr>
                         {{end}}
                     </tbody>
                 </table>
```

Comment:
I may be in the minority here, but I think that below both the "Latest Blocks" and "Mempool" tables it is worth having a "more" or "see all" kind of link, even though these links exist on the "Latest Blocks" and "Mempool" headers.